src/App/Styles/Base.xaml

```diff
@@ -97,13 +96,19 @@
         <Setter Property="TextColor"
                 Value="{StaticResource DisabledIconColor}" />
     </Style>
-    <Style TargetType="fab:FloatingActionButtonView">
-        <Setter Property="ColorNormal"
-                Value="{StaticResource FabColor}" />
-        <Setter Property="ColorPressed"
-                Value="{StaticResource FabPressedColor}" />
-        <Setter Property="ColorRipple"
-                Value="{StaticResource FabPressedColor}" />
+    <Style x:Key="floating-action-button" TargetType="Button">
```

Comment:
let's call the style `btn-fab` to be similar to other button style naming conventions.